python/ambassador/envoy/v2/v2listener.py

```diff
@@ -670,6 +670,9 @@ def __init__(self, config: 'V2Config', listener: IRListener) -> None:
         if 'xff_num_trusted_hops' in config.ir.ambassador_module:
             base_http_config["xff_num_trusted_hops"] = config.ir.ambassador_module.xff_num_trusted_hops
 
+        if 'idle_timeout' in config.ir.ambassador_module:
```

Comment:
This will indeed break basically every test out there, because the tests check for changes in the generated `envoy.json`. I think if you do

```
idle_timeout = config.ir.ambassador_module.get('idle_timeout', '0s')

if idle_timeout != '0s':
  ...
```

then you'll find that tests pass, except for the test you've surely added to change the `idle_timeout` and make sure it works? 🙂